eva-accession-core/src/test/java/uk/ac/ebi/eva/accession/core/test/fasta/FastaSequenceReaderTest.java

```diff
@@ -23,6 +23,8 @@
 import org.junit.rules.ExpectedException;
 import org.junit.rules.TemporaryFolder;
 
+import uk.ac.ebi.eva.accession.core.io.FastaSequenceReader;
```

Comment:
I think this import won't be needed if the test is moved to uk.ac.ebi.eva.accession.core.io package.